hibernate-envers/src/main/java/org/hibernate/envers/configuration/internal/AuditEntitiesConfiguration.java

```diff
@@ -98,6 +105,8 @@ public AuditEntitiesConfiguration(Properties properties, String revisionInfoEnti
 			revisionEndTimestampFieldName = null;
 		}
 
+        logAuditMappings = ConfigurationHelper.getBoolean(EnversSettings.LOG_AUDIT_MAPPINGS, properties, false);
```

Comment:
This is already handled by Envers inside [AdditionalJaxbMappingProducerImpl](https://github.com/hibernate/hibernate-orm/blob/master/hibernate-envers/src/main/java/org/hibernate/envers/boot/internal/AdditionalJaxbMappingProducerImpl.java#L71).  Basically if you want that dumped, simply enable trace logging for that class.  There is no need for a special configuration for this anymore.